HIP/hip-830.md

```diff
@@ -0,0 +1,350 @@
+---
+hip: 830
+title: Event Serialization With Round Number, EventDescriptors, and Multiple Other Parents.
+author: Edward Wertz <edward@swirldslabs.com>
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Last Call
+last-call-date-time: 2023-12-04T07:00:00Z
+created: 2023-10-25
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/827
+updated: 2023-12-04
+---
+
+## Abstract
+
+This HIP proposes several changes to the event serialization format.
+
+1. Add the `birthRound` to the metadata of new events. The `birthRound` value is 1 + the latest round to have come
+   to consensus at the time the event was created. This value is used to determine if the event is ancient or not.  
+   It is also used to look up the correct consensus roster for validating the event's signature.
+2. Represent references to parent events as `EventDescriptors`. EventDescriptors combine the existing data about each
+   parent (hash, creator, generation) and the birthRound into a single data structure.
+3. Allow for multiple other parents.
+4. In Gossip, reduce an event's unhashed metadata to just the event's signature.
+
+## Motivation
+
+### Add `birthRound` to Event Metadata
+
+The `birthRound` of an event is 1+ the latest round to have come to consensus at the time the event was created.  
+The `birthRound` is used to determine if an event is ancient or not and to look up the correct roster for
+validating the event's signature.
+
+### EventDescriptors as Parent Event References
+
+An EventDescriptor is a 4-tuple:
+
+1. `eventHash` - The hash of the event.
+2. `creatorId` - The NodeId of the node that created the event.
+3. `generation` - The generation of the event. (1 + the max of the event's parents' generations)
+4. `birthRound` - 1 + the latest round to reach consensus on the node when the event was created.
+
+The EventDescriptor is the minimal amount of information needed to identify an event: its hash, which node created
+it, and generation and birthRouns to determine when an event has become ancient. It is used in place of the event
```

Comment:
I would remove the sentence `It is used in place of the event itself when the payload of the event (the transactions) are not needed.` The parent event was never contained in the event, so this only describes usage in code, which is not the focus of this HIP, and could confuse someone who is not familiar with the code.